src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/clients/RoleAssignmentServiceApi.java

```diff
@@ -60,4 +61,11 @@ RoleAssignmentResource queryRoleAssignments(
         @RequestHeader(SERVICE_AUTHORIZATION) String s2sToken,
         @RequestBody MultipleQueryRequest queryRequest
     );
+
+    @GetMapping(
+        value = "/am/role-assignments/actors/{actorId}",
+        produces = "application/vnd.uk.gov.hmcts.role-assignment-service.get-assignments+json;charset=UTF-8;version=1.0"
+    )
+    RoleAssignmentResource getRolesByActorId(@PathVariable("actorId") String actorId,
+                                           @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthToken);
```

Comment:
This endpoint is the same as getRolesForUser()
Although it does not use a user token is this correct ?

Reply:
yes, that is correct.